services/create_ssl_certs.md

```diff
@@ -170,5 +173,22 @@ Submit the CSR to gandi.net. Use the following guidance:
       mv ${NEW_HOST_NAME}_chained.pem roles/nginxplus/files/ssl/
       ```
 
+#### Tigris
+
+In July of every year [tigris.princeton.edu](tigris.princeton.edu) will get an automatic renewal. The following steps will be needed to ensure the certificate remains renewed. You will need the [vaulted private key](https://github.com/pulibrary/princeton_ansible/blob/main/keys/tigris_princeton_edu_priv.key) and the certificate and intermediate certificate to generate a pfx file that you will ship to the vendor who hosts the application
+
+  ```bash
+  cat ~/path/to/downloads/tigris_princeton_edu_cert.cer ~/path/to/downloads/tigris_princeton_edu_interm.cer > keys/tigris_princeton_edu_chained.pem
+  ```
+
+This will generate a chained file. You will be prompted for a password in the next step.
+
+```bash
+  openssl pkcs12 -export -out tigris_princeton_edu.pfx -inkey tigris_princeton_edu_priv.key -in tigris_princeton_edu_chained.pem
+```
+
+Send the resulting file to the tigris support folks via [the Secure Send Portal](https://securesend.princeton.edu/#/) along with the password used above
```

Comment:
Maybe rewrite this with a first step to a open a ticket. I did this and then they gave me an individual email they wanted the securesend to go to. 

Suggestion - 

Open a ticket with tigris support at Gimmal Support support@gimmal.com and ask who should receive the new chained file. Send the resulting file to the email tigris support recommends to via [the Secure Send Portal](https://securesend.princeton.edu/#/) along with the password used above.

Reply:
Addressed by a2426f4554d2bdb6771503f5bf0cda56093b53ba.